### DISCUSSION TITLE
Molecule parameters only in simulation or also in building blocks?

### DISCUSSION THREAD

**Discussion by moPellowe** [Open-Systems-Pharmacology/Forum#1184] [Questions & Problems] [answered]
I transferred a simulation to MoBi, and I'm noticing that there are some parameters that show up under the molecule within the simulation but are not in the molecule building block (e.g. Ka (acidic phospholipids), Ka (proteins)). Are those parameters only found within the simulation or are they somewhere in the building blocks?

![image](https://user-images.githubusercontent.com/57540473/160300996-b6d58f93-7223-499b-ab71-aaaba61b59d0.png)
![image](https://user-images.githubusercontent.com/57540473/160301014-f4b8e188-9abe-4823-94b7-64b15c762427.png)

**Comment by Yuri05** [comment, marked as answer]
Those are supporting parameters, which are used only in some calculation methods (e.g. for partition coeffcients, cellular permeabilities etc.) and are stored in **C:\ProgramData\Open Systems Pharmacology\MoBi\X.Y\AllCalculationMethods.pkml**

![grafik](https://user-images.githubusercontent.com/25061876/161272664-38d31740-3bd2-4441-af30-13f260ad6d74.png)

So you will not find any of those parameter in the building blocks
Depending on the selection of different calculation methods for a molecule, supporting parameters are added during the simulation creation.

![grafik](https://user-images.githubusercontent.com/25061876/161273678-1018c303-9201-4e1b-b4e6-02bae3909976.png)
